tools/Nethermind.Tools.Kute/Auth/JwtAuth.cs

```diff
@@ -24,20 +23,20 @@ public JwtAuth(ISystemClock clock, ISecretProvider secretProvider)
         _clock = clock;
 
         var hexSecret = secretProvider.Secret;
-        _secret = Enumerable.Range(0, hexSecret.Length)
+        _key = new(Enumerable.Range(0, hexSecret.Length)
             .Where(x => x % 2 == 0)
             .Select(x => Convert.ToByte(hexSecret.Substring(x, 2), 16))
-            .ToArray();
+            .ToArray());
     }
 
     private string GenerateAuthToken()
     {
-        var signingKey = new SymmetricSecurityKey(_secret);
-        var credentials = new SigningCredentials(signingKey, SecurityAlgorithms.HmacSha256);
-        var claims = new[] { new Claim(JwtRegisteredClaimNames.Iat, _clock.UtcNow.ToUnixTimeSeconds().ToString(), ClaimValueTypes.Integer64) };
```

Comment:
We might be reintroducing https://github.com/NethermindEth/nethermind/pull/6936. @stdevMac please check again with Reth or by following that PR discussion.

Reply:
It's failing to compile:
```
MSBuild version 17.8.5+b5265ef37 for .NET
  Determining projects to restore...
  Restored /root/nethermind/tools/Nethermind.Tools.Kute/Nethermind.Tools.Kute.csproj (in 3.61 sec).
/root/nethermind/tools/Nethermind.Tools.Kute/JsonRpcMethodFilter/ComposedJsonRpcMethodFilter.cs(18,18): error CS1061: 'IEnumerable<IJsonRpcMethodFilter>' does not contain a definition for 'IsNullOrEmpty' and no accessible extension method 'IsNullOrEmpty' accepting a first argument of type 'IEnumerable<IJsonRpcMethodFilter>' could be found (are you missing a using directive or an assembly reference?) [/root/nethermind/tools/Nethermind.Tools.Kute/Nethermind.Tools.Kute.csproj]

Build FAILED.

/root/nethermind/tools/Nethermind.Tools.Kute/JsonRpcMethodFilter/ComposedJsonRpcMethodFilter.cs(18,18): error CS1061: 'IEnumerable<IJsonRpcMethodFilter>' does not contain a definition for 'IsNullOrEmpty' and no accessible extension method 'IsNullOrEmpty' accepting a first argument of type 'IEnumerable<IJsonRpcMethodFilter>' could be found (are you missing a using directive or an assembly reference?) [/root/nethermind/tools/Nethermind.Tools.Kute/Nethermind.Tools.Kute.csproj]
```

Reply:
@emlautarom1 The updated code has been tested to ensure it generates the same token as it was with the replaced package.